crud/common/utils.lua

```diff
@@ -524,4 +524,17 @@ function utils.flatten_obj_reload(space_name, obj)
     return schema.wrap_func_reload(flatten_obj, space_name, obj)
 end
 
+function utils.make_options(opts, add_space_schema_hash_value)
```

Comment:
It seems to me that the word ``create`` is more suitable here (not ``make``)

Reply:
`make_option` name is often used for parsing options (in Python modules for [example](https://docs.python.org/3/library/optparse.html#populating-the-parser)). Thats why it seems to me that this name is suitable.

Reply:
A bit more bikeshedding: it looks specific for the `add_space_schema_hash` option, so it would be better either choose a specific name or rewrite the function to be common:

```lua
-- Merge two options map.
--
-- `a` and/or `b` can be `nil`.
--
-- If `a.foo` and `b.foo` exists, prefer `b.foo`.
function utils.merge_options(a, b)
    return fun.chain(a or {}, b or {}):tomap()
end
```

So the call will be the following:

```diff
-opts = utils.make_options(opts, true)
+opts = utils.merge_options(opts, {add_space_schema_hash = true})
```

Maybe a bit more Lua tables, but the call becomes more readable (now we see what is the true/false value).

Reply:
Updated:

```diff
+-- Merge two options map.
+--
+-- `opts_a` and/or `opts_b` can be `nil`.
+--
+-- If `opts_a.foo` and `opts_b.foo` exists, prefer `opts_b.foo`.
+function utils.merge_options(opts_a, opts_b)
+    return fun.chain(opts_a or {}, opts_b or {}):tomap()
+end

```